group-generators/generators/sismo-early-users/index.ts

```diff
@@ -38,6 +38,8 @@ const generator: GroupGenerator = {
       {
         name: "sismo-early-users",
         timestamp: context.timestamp,
+        description: "",
```

Comment:
We should enter a description here. Is it possible to throw if the description is empty?